src/quo2/foundations/colors.cljs

```diff
@@ -1,17 +1,31 @@
 (ns quo2.foundations.colors
-  (:require [clojure.string :as string]
-            [quo2.theme :as theme]))
+  (:require [clojure.string :as string :refer [join replace split starts-with?]]
+            [quo.theme :as theme]))
```

Comment:
could you please rollback require? better to not use refer, what's the point here ? and also why quo2 theme was replaced by quo ?

Reply:
Thanks for finding this out, not sure why it quo2 theme require changed actually. Fixed it and rollbacked :)